torch_np/_wrapper.py

```diff
@@ -288,12 +288,15 @@ def arange(start=None, stop=None, step=1, dtype=None, *, like=None):
         raise ValueError("Maximum allowed size exceeded")
 
 
+@_decorators.dtype_to_torch
 def empty(shape, dtype=float, order="C", *, like=None):
     _util.subok_not_ok(like)
```

Comment:
There's a bit of a mismatch here?

Reply:
What mismatch?

Reply:
shouldn't we have a function that's  `_util.like_not_implemented(like)` or so? Why are we passing `like` to `subok`?

Reply:
Ah, so it's the util name which is confusing. Conceptually, I'd say that `like` and `subok` are similar: one is specifically for ndarray subclasses, the other one is for a newer `__array_wrap__` protocol (I think). Most functions which work for subclasses or the protocol, have both arguments, some have one or the other. We are not going to support either of them. The utility is literally two lines:
https://github.com/Quansight-Labs/numpy_pytorch_interop/blob/main/torch_np/_detail/_util.py#L22

So if we go with utility functions I'd keep it as is, can rename it if really wanted. If we transform to the normalization approach of gh-32, these two would likely get their own type annotations. 

So overall I'd rather not touch it now.